user/tlu/module/src/TluRawEvent2StdEventConverter.cc

```diff
@@ -131,6 +131,14 @@ bool TluRawEvent2StdEventConverter::Converting(eudaq::EventSPC d1, eudaq::Standa
   d2->SetTimeEnd(d1->GetTimestampEnd() * 1000);
   d2->SetTimestamp(ts, d1->GetTimestampEnd(), d1->IsFlagTimestamp());
 
+  // print time stamps to have a look
```

Comment:
Guess that's a leftover from debugging? :)

Reply:
yeth, not sure how this ended up here ;)